.github/container/test-pax.sh

```diff
@@ -90,7 +96,8 @@ done
 
 # # Set derived variables
 
-NGPUS=$(nvidia-smi -L | grep -c '^GPU')
+GPUS_PER_NODE=$(nvidia-smi -L | grep -c '^GPU')
+NGPUS=$(( $GPUS_PER_NODE * $NODES ))
```

Comment:
`$((GPUS_PER_NODE * NODES))` should do since we are already using `$((...))`. There is no need for the extra dollar signed in front of individual variables.